Lib/asyncio/tasks.py

```diff
@@ -42,9 +42,19 @@ def all_tasks(loop=None):
     """Return a set of all tasks for the loop."""
     if loop is None:
         loop = events.get_running_loop()
-    # NB: set(_all_tasks) is required to protect
+    # NB: list(_all_tasks) is required to protect
     # from https://bugs.python.org/issue34970 bug
```

Comment:
Can we write a proper comment here without referencing an issue on bpo?  I mean a url is fine, but it requires the reader to navigate to it in order to understand what's going on.